study.md

```diff
@@ -57,7 +63,9 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+Yes it creates the objects and saves into the database.
+person = Person.create([{name: "Pinaro"}, {last_name : 'Ouk'}])
+http://api.rubyonrails.org/classes/ActiveRecord/Base.html
 ```
```

Comment:
check out the create syntax. This will currently create two records. One with a name and one with a last name